Find-and-Replace/Replace-line-break-with-paragraph-mark/.NET-Standard/Replace-line-break-with-paragraph-mark/Program.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Syncfusion.DocIO;
+using Syncfusion.DocIO.DLS;
+using System.IO;
+
+namespace Replace_line_break_with_paragraph_mark
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            using (FileStream fileStreamPath = new FileStream(Path.GetFullPath(@"../../../Data/Input.docx"), FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
+            {
+                //Open an existing Word document.
+                using (WordDocument document = new WordDocument(fileStreamPath, FormatType.Docx))
+                {
+                    // Iterate through the paragraphs of Word document.
+                    foreach (WParagraph paragraph in document.Sections[0].Body.Paragraphs)
+                    {
+                        for (int i = 0; i < paragraph.Items.Count; i++)
```

Comment:
Can you check iterating elements in UG and modify the sample as generic requirement.

Sample should replace line break in all areas of document para, table, textbox, content control everything

Reply:
Modified